src/Microsoft.DotNet.Darc/src/Darc/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
```

Comment:
Hrm... it seems like this file should be excluded with .gitignore and not checked in.  One day someone is going to debug with a secret and we'll be sad.

Reply:
I have tried like 30 times to get this thing excluded...i wonder if it needs to be removed from the index, then .gitignored

Reply:
Yeah. Once it's checked in, changing the gitignore won't automatically remove it.  You should be able to exclude it then run "git rm" on it.  It shouldn't get picked back up after that.